src/Automation/Factory.cs

```diff
@@ -8,6 +8,14 @@ namespace Axe.Windows.Automation
 {
     static class Factory
     {
+        static public IScanTools CreateScanTools()
+        {
+            return new ScanTools(
+                CreateOutputFileHelper(null),
```

Comment:
>null [](start = 39, length = 4)

How does the Config's OutputDirectory property override the null that is getting set by default? It feels like maybe the CreateScanTools method should accept an IConfig parameter

Reply:
In changes that are already made and coming down the pipe, the Config is set to a parameter passed into the factory method.